bundle/edu.gemini.spModel.pio/src/main/java/edu/gemini/spModel/pio/Pio.java

```diff
@@ -4,6 +4,7 @@
 package edu.gemini.spModel.pio;
 
 import scala.Tuple2;
+import scalaz.Alpha;
```

Comment:
Wiz zat?  Doesn't seem to be used?

Reply:
Huh... No idea how that got in there, or even what it does. Removed.